src/sensors/CyclePowerData.cpp

```diff
@@ -62,11 +57,13 @@ void CyclePowerData::decode(uint8_t *data, size_t length) {
   if (bitRead(flags, 5)) {
     // Crank Revolution data present, lets process it.
     this->lastCrankRev       = this->crankRev;
-    this->crankRev           = bytes_to_int(data[cPos + 1], data[cPos]);
+    this->crankRev           = bytes_to_u16(data[cPos + 1], data[cPos]);
     this->lastCrankEventTime = this->crankEventTime;
-    this->crankEventTime     = bytes_to_int(data[cPos + 3], data[cPos + 2]);
-    if (this->crankRev > this->lastCrankRev && this->crankEventTime - this->lastCrankEventTime != 0) {
-      int cadence = ((abs(this->crankRev - this->lastCrankRev) * 1024) / abs(this->crankEventTime - this->lastCrankEventTime)) * 60;
+    this->crankEventTime     = bytes_to_u16(data[cPos + 3], data[cPos + 2]);
+    if (this->crankRev > this->lastCrankRev && this->crankEventTime != this->lastCrankEventTime) {
+      float crankChange = abs(this->crankRev - this->lastCrankRev) * 1024;
```

Comment:
Might be worth marking `crankChange` and `timeElapsed` as `const`.  Thoughts?

Reply:
No objection there

Reply:
@kadaan , feel free to make this change and merge.